src/infra/redis/ioredis.adapter.ts

```diff
@@ -0,0 +1,197 @@
+import { Redis } from 'ioredis';
+import { Logger } from '../logger/index.js';
+import { addMessageCommand, xDelIfEmptyCommand } from './commands.js';
+import { Task, XAutoClaimResponse } from './interfaces/redis.interface.js';
```

Comment:
Ist es eine bewusste Entscheidung mit mit barrel file zu arbeiten?
import { Task, XAutoClaimResponse, StreamMessageReply, StreamsMessagesReply,  StreamNameClockPair  } from './interfaces';

Reply:
Fixed by adding barrel to `src/infra/redis/interfaces`!